packages/FeedbackIcon/FeedbackIcon.jsx

```diff
@@ -25,10 +31,16 @@ const FeedbackIcon = ({ copy, copyDictionary, optionalText, children, ...rest })
   }
 
   return (
-    <svg {...safeRest(rest)} role="img" aria-hidden={a11yText === '' ? true : undefined}>
+    <StyledSVG
+      {...safeRest(rest)}
+      role="img"
+      aria-hidden={a11yText === '' ? true : undefined}
+      width={`${width / 16}rem`}
+      height={`${height / 16}rem`}
```

Comment:
what is the the significance of 16 here? Can we store that as a const `MAGIC_NUMBER = 16` (with a more descriptive variable name of course) on the top of the file and use it here?

Reply:
16px is the default font size in almost all browsers, and TDS assumes this. This pixel-to-rem conversion could be something that we move into sometime like`@tds/shared-utils` as it is currently being used in a couple other different places.